### PR TITLE
Redirect /firefox/mobile/get-app/ to /firefox/browsers/mobile/get-app/

### PR DESCRIPTION
**Pull request by Soumya0803**
## One-line summary

The [get-app page](https://www.mozilla.org/en-US/firefox/mobile/get-app/) should live under the same IA as the rest of the Firefox mobile pages and redirect the old URL.

## Significant changes and points to review
 - Page now lives at new URL
 - Redirect is in place for old URL

## Issue / Bugzilla link
#13469 

## Testing
http://localhost:8000/en-US/firefox/browsers/mobile/get-app/ 
http://localhost:8000/en-US/firefox/mobile/get-app/

### CODE REVIEW

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -557,4 +557,5 @@ def firefox_channel(*args, **kwargs):
     redirect(r"^firefox/sync/?$", "firefox.features.sync"),
     # issue 13732
     redirect(r"^firefox/welcome/3/?$", "firefox.accounts"),
+    redirect(r"^firefox/mobile/get-app/", "https://www.mozilla.org/en-US/firefox/browsers/mobile/get-app/"),
```
Comment (alexgibson) [inline bedrock/firefox/redirects.py:560]:
```suggestion
    redirect(r"^firefox/mobile/get-app/", "firefox.browsers.mobile.get-app"),
```

Comment (alexgibson) [reply bedrock/firefox/redirects.py:560]: Can you please also add a test for this redirect at the bottom of the following file?

https://github.com/mozilla/bedrock/blob/main/tests/redirects/map_globalconf.py

**Review by alexgibson — CHANGES_REQUESTED** [review]
Thanks so much for this, great work!

Just one small change and it would be great to add a test for the redirect? r+wc!

File: bedrock/firefox/urls.py [hunk 2]
```diff
@@ -208,6 +207,8 @@
         "firefox/browsers/mobile/compare.html",
         ftl_files=["firefox/browsers/mobile/compare", "firefox/browsers/compare/shared"],
     ),
+    page("firefox/browsers/mobile/get-app/", "firefox/browsers/mobile/get-app.html", ftl_files=["firefox/mobile"]),
+
```
Comment (alexgibson) [inline bedrock/firefox/urls.py:211]: Oh, it looks like our linter also wants this blank line to be removed. If you do that then the CI check here in the pull request should be happy again.

**Comment by Soumya0803** [comment]
Thanks for the review @alexgibson. I have added the requested changes.

**Review by alexgibson — APPROVED** [review]
r+

thanks for contributing! 🎉

**Pull request merged**